scripts/create-certs.sh

```diff
@@ -40,4 +49,4 @@ openssl ecparam -name prime256v1 -genkey -noout -out expired_leaf_key.pem
 openssl req -new -key expired_leaf_key.pem -out expired_leaf_csr.pem
 
 # Sign the Leaf certificate with the Intermediate
-openssl x509 -req -in expired_leaf_csr.pem -CA intermediate_cert.pem -CAkey intermediate_key.pem -out expired_leaf_cert.pem -days -365
+openssl x509 -req -in expired_leaf_csr.pem -CA intermediate_cert.pem -CAkey intermediate_key.pem -out expired_leaf_cert.pem -days -1
```

Comment:
I forgot to add at the end:
```suggestion
openssl x509 -req -in expired_leaf_csr.pem -CA intermediate_cert.pem -CAkey intermediate_key.pem -out expired_leaf_cert.pem -days -1

popd
```